chart/compass/charts/external-services-mock/templates/oauth-credentials-secret.yaml

```diff
@@ -50,12 +50,11 @@ metadata:
   namespace: compass-system
 type: Opaque
```

Comment:
can we move this secret to be not between ext-svc-mock-oauth-secrets(kyma/compass ns) and if needed to add manage property dedicated to this secret because now it uses the one from ext-svc-mock?